internal/cmd/skupper/site/nonkube/site_create.go

```diff
@@ -48,12 +46,10 @@ func (cmd *CmdSiteCreate) NewClient(cobraCommand *cobra.Command, args []string)
 }
 
 func (cmd *CmdSiteCreate) ValidateInput(args []string) []error {
-
 	var validationErrors []error
-
-	if cmd.Flags.ServiceAccount != "" {
```

Comment:
Why is this warning deleted?

Reply:
looks like a mistake.   I'll add it back